examples/README.md

```diff
@@ -0,0 +1,5 @@
+# DeepReg Examples
+
+This folder contains multiple self-contained example scripts for certain DeepReg
+functionalities. The purpose is mainly to showcase some functionalities like using
```

Comment:
certain -> customising (this is the only one we are showcasing for now, if others, we can change the wording here)
like -> such as (a bit more formal)